cmd/zetae2e/config/local.yml

```diff
@@ -65,6 +65,10 @@ additional_accounts:
     bech32_address: "zeta1nry9yeg6njhjrp2ctppa8558vqxal9fxk69zxg"
     evm_address: "0x98c852651A9CAF2185585843d3D287600Ddf9526"
     private_key: "bf9456c679bb5a952a9a137fcfc920e0413efdb97c36de1e57455763084230cb"
+  user_emissions_withdraw:
```

Comment:
Isn't one of the observer? I would rather call it `user_observer` (or `observer_hot_key` to be more precise), might be used for other purpose in the future

Reply:
> ; it's a new account used just for withdrawal.

Maybe then call it user_emissions_receiver?

Reply:
it's, not the receiver; the operator is still the receiver . 
It just executed the withdraw tx